### ISSUE TITLE
Cloning the same private repo under different names freezes the GUI

### ISSUE THREAD

**Issue by EmilyYVChan** [kblincoe/VisualGit_SE701_3#71]
Bug report:
When you try to clone the same private repo a second time with a different file location, the GUI blacks out and freezes. Also, the directory (for the different file location) is created, but it is empty. 

To reproduce this bug:
1. Clone a private repo to your local machine.
2. Clone the same private repo again, but under a different file location in your local machine. 
(before doing step 2, don't delete the cloned repo from step 1).

Cloning the private repo for the first time, which VisualGit is able to do so successfully.
![image](https://user-images.githubusercontent.com/22021797/37493511-0521c858-290a-11e8-843e-46023a40fa29.png)
![image](https://user-images.githubusercontent.com/22021797/37493525-14bfa5dc-290a-11e8-9432-958dc7bbfc4a.png)

Attempting to clone the same private repo for the second time, but unsuccessfully.
![image](https://user-images.githubusercontent.com/22021797/37493540-28300ba2-290a-11e8-9963-098c950ddfd5.png)
![image](https://user-images.githubusercontent.com/22021797/37493553-327cd40a-290a-11e8-9433-a51413f05c88.png)

Contents of the relevant directories
![image](https://user-images.githubusercontent.com/22021797/37493580-4b2c9382-290a-11e8-845d-68ccc6865336.png)
![image](https://user-images.githubusercontent.com/22021797/37493951-aaa09cea-290b-11e8-8ccb-135318751cff.png)

@softeng-701 for review

**Comment by softeng-701** [comment]
@EmilyYVChan Approved

**Comment by sguy016** [comment]
Claimed

**Comment by sguy016** [comment]
@EmilyYVChan I can not  reproduce the problem. Do you still have this issue ?

**Comment by EmilyYVChan** [comment]
@sguy016 Yup I still have this issue. I have just reproduced the problem in the same way as per the description above. For additional info, I am running VisualGit on a Windows 10 machine.

**Comment by EmilyYVChan** [comment]
@sguy016  as requested, I have tried to do the same thing with a different repo. Kindly see the screenshots below:

![image](https://user-images.githubusercontent.com/22021797/37880071-68c45150-30df-11e8-8696-e69175ccbb0f.png)
![image](https://user-images.githubusercontent.com/22021797/37880167-eedcd45a-30e0-11e8-9990-ff29a0d4c482.png)
![image](https://user-images.githubusercontent.com/22021797/37880177-f871a428-30e0-11e8-90d1-eeca653b031e.png)
![image](https://user-images.githubusercontent.com/22021797/37880181-06418a28-30e1-11e8-9fa5-a0ad2442578a.png)
![image](https://user-images.githubusercontent.com/22021797/37880182-0e1cf778-30e1-11e8-856b-82d87c4c2032.png)

**Comment by EmilyYVChan** [comment]
@sguy016 I discovered that this bug only applies for private repositories. For public repositories it works fine. I have updated the original description to reflect this discovery. Thanks!
